Legacy/src/main/java/com/infomaniak/lib/core/githubTools/GitHubViewModel.kt

```diff
@@ -48,3 +51,9 @@ class GitHubViewModel : ViewModel() {
         const val API_GITHUB_URL = "https://api.github.com/repos/Infomaniak/"
     }
 }
+
+
+@Suppress("RedundantSuspendModifier")
+private suspend inline fun <T> Result<T>.cancellable(): Result<T> = onFailure {
+    if (it is CancellationException) throw it
+}
```

Comment:
No need to put the entire function in this file, just `if (it is CancellationException) throw it` in the on failure
Also you have two blank lines here